packages/core/adder/options.ts

```diff
@@ -37,8 +37,9 @@ export type OptionValues<Args extends OptionDefinition> = {
             : never;
 };
 
-export type AvailableCliOptionKeys = "path" | "skipPreconditions" | "skipInstall";
+export type AvailableCliOptionKeys = keyof AvailableCliOptionKeyTypes;
```

Comment:
Simplified this so we don't have to declare the names twice

Reply:
I knew I was missing something while creating those types. Thanks for catching that.